profile/README.md

```diff
@@ -1,14 +1,19 @@
 <h1 align="center">EddieHub</h1>
 
-<p align="center">
-Most inclusive Open Source community on the planet!
-</p>
-
 <h2 align="center">Collaboration 1st, Code 2nd! </h2>
 <p align="center">
-<img src="https://github.com/EddieHubCommunity.png" align="center" height="200" />
+    <img src="https://github.com/EddieHubCommunity.png" align="center" height="200" />
+    <h6 align="center"> Meet Hubber! Our Mascot! 🤝 </h6>
 </p>
 
+Here, right now on **your** screen, you are in the presence of the most inclusive *Open Source* community on the planet!
+
+
+This organisation is the home of *open source*! Make Pull Requests, open Issues, share ideas, help each other, fix problems! 
+
+## This is a place welcome to all!
```

Comment:
I would suggest centering those lines

Reply:
If i center those, it would require HTML code, is that OK? @schmelto

Reply:
Yes. I think we should only use html rgas in here